loader/src/consumer_class.py

```diff
@@ -68,7 +71,10 @@ def consume_event(self):
                 self.manage_error(msg)
             else:
                 # Proper message
-                self.process_event(msg)
+                try:
+                    self.process_event(msg)
+                except Exception as err:
+                    logger.error(f"failed to process kafka event: {err}")
```

Comment:
I would have put the try/except in the definition of `process_event`. There is already a try/except there, we could just make it englobe everything and except different types of errors.
What do you think?

Reply:
yeah I agree, it's probably cleaner this way